fullscreen.bs

```diff
@@ -102,7 +105,7 @@ it from its <a>node document</a>'s <a>top layer</a>.
  that have their <a>fullscreen flag</a> set, in <a>shadow-including tree order</a>.
 
  <li>
-  <p>For each <var>node</var> in <var>nodes</var>, run these substeps:
+  <p><a>For each</a> <var>node</var> in <var>nodes</var>, run these substeps:
```

Comment:
Since only "for each" is used now, I've gone with "type:dfn; for:set; text:for each" to make the source a bit more readable.